pkg/health/condition/builder.go

```diff
@@ -23,6 +23,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+//
+var skipMergeConditions = map[druidv1alpha1.ConditionType]bool{
```

Comment:
```suggestion
var skipMergeConditions = map[druidv1alpha1.ConditionType]struct{}{
```
Since the value doesn't matter.

Reply:
Thanks for pointing this out :) 
Updated